apps/settings/src/components/AppList.vue

```diff
@@ -48,19 +65,24 @@
 
 			<transition-group v-if="useBundleView"
 				name="app-list"
-				tag="div"
+				tag="table"
 				class="apps-list-container">
 				<template v-for="bundle in bundles">
-					<div :key="bundle.id" class="apps-header">
-						<div class="app-image" />
-						<h2>{{ bundle.name }} <input type="button" :value="bundleToggleText(bundle.id)" @click="toggleBundle(bundle.id)"></h2>
-						<div class="app-version" />
-						<div class="app-level" />
-						<div class="app-groups" />
-						<div class="actions">
-							&nbsp;
-						</div>
-					</div>
+					<tr :key="bundle.id" class="apps-header">
+						<th class="app-image">
+							<span class="hidden-visually">{{ t('settings', 'App icon') }}</span>
+						</th>
+						<th><h2>{{ bundle.name }} <input type="button" :value="bundleToggleText(bundle.id)" @click="toggleBundle(bundle.id)"></h2></th>
```

Comment:
Wouldn't something like that be better?

```suggestion
						<th colspan="4"><h2>{{ bundle.name }} <input type="button" :value="bundleToggleText(bundle.id)" @click="toggleBundle(bundle.id)"></h2></th>
```

Reply:
i've tried it out and that makes no difference. Actually it is only a name of the bundle available and nothing else.

Reply:
Thinking about it further, accessibility wise, won't this be confusing to have the bundle name in the app name column ?

Maybe we should have a table per bundle ? Like we did in timelines for Photos, one list per month.

Reply:
i get your point. I've checked it before and (actually) multiple headers should be fine https://www.w3.org/WAI/tutorials/tables/multi-level/

Reply:
i mean it is nicer solution with different tables but multiple should be fine too

Reply:
@Pytal @susnux what do you mean?

Reply:
> Thinking about it further, accessibility wise, won't this be confusing to have the bundle name in the app name column ?
> 
> Maybe we should have a table per bundle ? Like we did in timelines for Photos, one list per month.

I agree, `bundle.name` should not be nixed with column titles.

It should be either another heading level or captions of separated tables. And with bundles I think separated tables looks good, simpler and easier to use by voice (you can go one table to next one).

Reply:
> Thinking about it further, accessibility wise, won't this be confusing to have the bundle name in the app name column ?

Good point! But there are even more problems `input type="button" ` https://github.com/nextcloud/server/blob/f44a0a6dcc38238bc044072088804d179422b7a8/apps/settings/src/components/AppList.vue#L75 🙈

Reply:
i will make an implementation like here: example2, Summaries for more complex tables, https://www.w3.org/WAI/tutorials/tables/caption-summary/#:~:text=A%20caption%20functions%20like%20a,primary%20mechanism%20to%20identify%20tables.

Reply:
done